config/sync/core.entity_view_display.node.news.diff.yml

```diff
@@ -0,0 +1,123 @@
+uuid: 8eb371c4-2d92-47c4-8800-89a3e24389f9
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.diff
+    - field.field.node.news.body
+    - field.field.node.news.field_meta_tags
+    - field.field.node.news.field_photo
+    - field.field.node.news.field_published_date
+    - field.field.node.news.field_summary
+    - field.field.node.news.field_teaser
+    - field.field.node.news.field_toc_enable
+    - node.type.news
+  module:
+    - datetime
+    - field_group
+    - layout_builder
+    - media
+    - metatag
+    - text
+    - user
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: false
+  field_group:
+    group_additional_content_details:
+      children:
+        - field_teaser
+        - field_toc_enable
+        - field_meta_tags
+      parent_name: ''
+      weight: 6
+      format_type: html_element
+      region: content
+      format_settings:
+        id: ''
+        classes: field-collection
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+      label: 'Additional content details'
+id: node.news.diff
+targetEntityType: node
+bundle: news
+mode: diff
+content:
+  body:
+    label: hidden
+    type: text_default
+    weight: 5
+    settings: {  }
+    third_party_settings: {  }
+    region: content
+  content_moderation_control:
+    weight: 0
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_meta_tags:
+    type: metatag_empty_formatter
+    weight: 10
+    region: content
+    label: inline
+    settings: {  }
+    third_party_settings: {  }
+  field_photo:
+    weight: 4
+    label: hidden
+    settings:
+      image_style: ''
+      image_link: ''
+    third_party_settings: {  }
+    type: media_thumbnail
```

Comment:
@omahm left a question about this type in https://github.com/dof-dss/nidirect-drupal/pull/249#discussion_r392096314 - does it need to be `entity_reference_label` or okay as `media_thumbnail`?

Reply:
I think media_thumbnail is OK for now - but later it will need to change to a responsive image style for news images.